core/domain/html_validation_service.py

```diff
@@ -703,3 +703,31 @@ def is_parsable_as_xml(xml_string):
         return True
     except xml.etree.ElementTree.ParseError:
         return False
+
+
+def convert_svg_diagram_tags_to_image_tags(html_string):
+    """Renames all the oppia-noninteractive-svgdiagram on the
+       server to oppia-noninteractive-image and changes
+       corresponding attributes
+
+    Args:
+        html_string: str. The HTML string to check.
+
+    Returns:
+        str. The updated html string.
+    """
+    soup = bs4.BeautifulSoup(
+        html_string.encode(encoding='utf-8'), 'html.parser')
+
+    for image in soup.findAll(name='oppia-noninteractive-svgdiagram'):
+        # The 'filepath-with-value' and 'alt-with-value'
+        # attribute values should be enclosed in
+        # double quotes(&amp;quot;).
+        attr_value = escape_html(image['svg_filename-with-value'])
```

Comment:
Can you confirm that the SVG filename for svgdiagram is also stored in the same GCS buckets as standard noninteractive-images?

Reply:
yes both the buckets are same, I confirmed from the network calls made to fetch both. The paths are the same.

Reply:
Thanks--that's important to check.